src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -83,14 +104,32 @@ public JedisCluster(HostAndPort node, final JedisClientConfig clientConfig, int
     this(Collections.singleton(node), clientConfig, maxAttempts, poolConfig);
   }
 
+  /**
+   * Creates a Jedis-Cluster with multiple Nodes/Instances.
+   * Here, the default Timeout of {@value #DEFAULT_TIMEOUT} ms is being used with {@value  #DEFAULT_MAX_ATTEMPTS} maximum attempts.
+   * @param nodes Hosts to connect to.
+   */
   public JedisCluster(Set<HostAndPort> nodes) {
     this(nodes, DEFAULT_TIMEOUT);
   }
 
+  /**
+   * Creates a Jedis-Cluster with multiple Nodes/Instances.
```

Comment:
Please don't use `Jedis-Cluster`. You can use `JedisCluster` or `Cluster` or `cluster`.